src/routes.js

```diff
@@ -28,8 +46,12 @@ routes.info = function (req, res, db) {
 }
 
 routes.renderNetwork = function (network, res, db) {
-  db.Pod.projectStats(network, function (projectStats) {
```

Comment:
@Flaburgan I think you were supposed to remove this from database.js by looking at the commit - but it's still there? :)